lib/index.js

```diff
@@ -85,7 +85,7 @@ internals.implementation = function (server, options) {
           // Authenticated
 
           return reply.continue({ credentials: credentials });
-        });
+        }, request);
```

Comment:
I believe your looking for http://hapijs.com/api#replycontinuedata?  The above code appears to be passing the request object on to the custom validate function not reply.continue.  So the custom validation function signature would be yourFunctionCallback(decodedToken, callback, request)